darts/models/forecasting/regression_model.py

```diff
@@ -570,16 +571,34 @@ def predict(
             They must match the covariates used for training in terms of dimension and type.
         num_samples : int, default: 1
             Currently this parameter is ignored for regression models.
+        likelihood_parameters : bool, default : False
+            Forecast contains likelihood model parameters instead of the target values, `num_samples` must be 1. Only
+            available for `LinearRegressionModel`, `LGBMModel`, `CatboostModel`, and `XGBoostModel`.
         **kwargs : dict, optional
             Additional keyword arguments passed to the `predict` method of the model. Only works with
             univariate target series.
         """
+        # Test is already performed by ForecastingModel.predict(), called by GlobalForecastingModel.predict()
         raise_if(
             not self._is_probabilistic() and num_samples > 1,
             "`num_samples > 1` is only supported for probabilistic models.",
             logger,
         )
 
+        raise_if(
+            likelihood_parameters and not isinstance(self, _LikelihoodMixin),
+            f"{self.__class__} does not support `likelihood_parameters=True` as it does not inherit from "
+            f"`_LikelihoodMixin`. The models supporting it are `LinearRegressionModel`, `LGBMModel`, "
+            f"`CatboostModel` and `XGBoostModel`.",
```

Comment:
we got to check also that the model is probablistic

```suggestion
            f"`likelihood_parameters=True` is only supported for probabilistic regression models.",
```